sectionctl_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/url"
+	"path/filepath"
+	"testing"
+
+	"github.com/alecthomas/kong"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBootstrapSetsUpDebugLogFile(t *testing.T) {
+	assert := assert.New(t)
+
+	// Setup
+	u, err := url.Parse("https://aperture.section.io")
+	assert.NoError(err)
+	d, err := ioutil.TempDir("", "sectionctl")
+	assert.NoError(err)
+
+	var cli = CLI{Debug: true, SectionToken: "s3cr3t", SectionAPIPrefix: u, DebugFileDir: d}
+	var ctx = kong.Context{}
```

Comment:
(nitpick) same as the default value:

```go
var ctx kong.Context
```

Reply:
Good call. It was being initialized with a value, but I refactored it so it didn't.